libs/goldbach.js

```diff
@@ -12,30 +12,74 @@ class goldbachFormulas {
   /**
   * @property {Function} set_max_number sets the number we're looking to find
   *           all additions for
-  * @param number the number we're going to process
+  * @param {number} number the number we're going to process
   */
-  set_max_number(number){
+  set_max_number(number) {
     this.__max_number = number;
   }
 
   /**
    * @property {Function} run_process executes the goldbach formulas in the
    *           class
+   * @return {function} goldbach formula
    */
-  run_process(){
-    console.log('Call other goldbach methods against ' + this.__max_number +
-        ' and output the result'
-    );
+  run_process() {
+    return this.__goldbach_formula();
   }
 
-  __get_primes(){
+  /**
+   * @property {Function} __get_primes executes the goldbach formulas in the
+   *           class
+   *  @param {int} n is prime reference
+   *  @return {int} primes
+   */
+  __get_primes(n) {
     // reference this.__max_number
+    const x = [];
+    let i = 0;
+    let j = 0;
+    const primes = [];
+    for (i = 2; i <= n; ++i) {
+      if (!x[i]) {
+        primes.push(i);
+        for (j = i << 1; j <= n; j += i) {
+          x[j] = true;
+        }
+      }
+    }
+    return primes;
   }
 
-  __goldbach_formula(){
+
+  /**
+* @property {Function} __goldbach_formula executes the goldbach formulas in the
+*           class
+*
+*@return {tupple} return tupples
+*/
+  __goldbach_formula() {
     // reference this.__max_number
-  }
 
+    const primes = this.__get_primes(this.__max_number);
```

Comment:
Once the method for __get_primes is updated, remove the argument of this.__max_number from this line.

```suggestion
    const primes = this.__get_primes();
```